nodeJS/APIs/APIs.md

```diff
@@ -55,6 +58,15 @@ For now, it is acceptable to just allow access from any origin. This makes devel
 </div>
 
 ### Additional Resources
+
 This section contains helpful links to other content. It isn't required, so consider it supplemental.
 
 - A [simple](https://simple.wikipedia.org/wiki/Representational_state_transfer) example based definition of REST.
+
+### Knowledge Check
+
+- <a class='knowledge-check-link' href='#rest'>What does REST stand for?</a>
+- <a class='knowledge-check-link' href='#rest'>What are HTTP verbs and why are they important to an API?</a>
+- <a class='knowledge-check-link' href='#cors'>What is the Same-Origin Policy?</a>
+- <a class='knowledge-check-link' href='#cors'>How do you enable CORS in your Express app?</a>
+- <a class='knowledge-check-link' href='#assignment'>Which HTTP verb does each letter in CRUD (Create, Read, Update, Delete) correspond to?</a>
```

Comment:
```suggestion
- <a class='knowledge-check-link' href='https://www.robinwieruch.de/node-express-server-rest-api/'>Which HTTP verb does each letter in CRUD (Create, Read, Update, Delete) correspond to?</a>
```
Linking directly to the resource being referenced, rather than just to the assignment section, is a better user experience for these IMO.